### PR TITLE
Tag Nemo v0.4.1

### PR DESCRIPTION
**Pull request by wbhart**

### CODE REVIEW

File: Nemo/versions/0.4.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4-
```
Comment (tkelman) [inline Nemo/versions/0.4.1/requires]: the trailing `-` refers to pre-0.4.0 prereleases, do you still need to support those?

**Comment by tkelman** [comment]
Do you know why your tests are failing on Travis CI? The PPA method is also not recommended, `language: julia` is the preferred method now and would allow you to also test against osx.

**Comment by wbhart** [comment]
We don't need the prerelease versions any more I guess.

I have not set up Travis for Nemo. I've only had time to set it up for some
of the other projects I manage.

**Comment by tkelman** [comment]
It's running, just not in a way that succeeds: https://travis-ci.org/wbhart/Nemo.jl/builds/108891604

**Comment by wbhart** [comment]
Well I've never set it up. It looks to me like it is trying to test Nemo
without building it.

**Pull request merged**